### PR TITLE
Fix for using multiple flags with adop compose init

### PR DESCRIPTION
**Pull request by statlus**
Wrapped the case statement at the start of adop compose init() with a while loop so all options are read instead of just the first one.

### CODE REVIEW

File: cmd/compose [hunk 1]
```diff
@@ -86,18 +86,25 @@ prep_env() {
...
+        --without-load)
+            export LOAD="NO"
+            shift
+            ;;
+        --with-stdout)
+            export LOGS="NO"
+            shift
+            ;;
+        *)
+            shift
```
Comment (nickdgriffin) [inline cmd/compose]: Was this added to prevent an infinite loop from occurring?

**Comment by statlus** [comment]
I've changed the catchall option to echo a message and exit 1.

**Comment by nickdgriffin** [comment]
Finally got around to testing this, and it's all dandy from Ubuntu 16.04. I'll give it a shot from Windows too soon.

@anton-kasperovich Would you be able to give it a quick go at some point on OSX please?

**Comment by nickdgriffin** [comment]
Tested from Windows 7 with the latest version of Docker Toolbox and it works fine.

@statlus Could you rebase on master just so the history is tidier please?

**Comment by anton-kasperovich** [comment]
Tested from macOS Sierra (10.12.3) and it works. Thank you!

**Comment by statlus** [comment]
Hi @nickdgriffin, I've rebased the change on master.

**Comment by nickdgriffin** [comment]
Great, thanks.

So, LGTM!

**Pull request merged**